client/my-sites/stats/info-panel/index.jsx

```diff
@@ -86,7 +86,7 @@ class StatsInfoPanel extends React.PureComponent {
 									href="http://wordpress.com/support/stats/#clicks"
 								>
 									<Gridicon icon="info-outline" />
-									{ this.props.translate( 'About Clicks' ) }
+									{ this.props.translate( 'About clicks' ) }
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.